docs/image.md

```diff
@@ -505,16 +505,17 @@ Does not work for static image resources.
 ### `prefetch()`
 
 ```jsx
-Image.prefetch(url);
+await Image.prefetch(url);
 ```
 
-Prefetches a remote image for later use by downloading it to the disk cache
+Prefetches a remote image for later use by downloading it to the disk cache. Returns a promise which resolves to a boolean.
 
 **Parameters:**
 
-| Name | Type   | Required | Description                       |
-| ---- | ------ | -------- | --------------------------------- |
-| url  | string | Yes      | The remote location of the image. |
+| Name       | Type     | Required | Description                                                         |
+| ---------- | -------- | -------- | ------------------------------------------------------------------- |
+| url        | string   | Yes      | The remote location of the image.                                   |
+| callback   | function | No       | [Android only] The function that will be called with the requestId. |
```

Comment:
Can you use the platform label for marking the Android only parameter like on the Modal page? 

https://github.com/facebook/react-native-website/blame/master/docs/alert.md#L211

Reply:
I used the android label android like on the Alert Page, not sure how this will look.

Reply:
Deploy failed because there is a syntax error on the page. It looks like not the whole label HTML has been added.

Reply:
> not sure how this will look

You can check how label looks on the Alert page: https://reactnative.dev/docs/next/alert#methods

Reply:
thanks for the help, I fixed it